biomage/stage/stage.py

```diff
@@ -305,6 +313,22 @@ def create_manifest(templates, token, repo_to_ref, all_experiments, auto):
         click.echo("\n".join(f"• {pin}" for pin in pins))
     else:
         click.echo("Not pinning any repository")
+
+    if not auto:
```

Comment:
What is auto?

Reply:
it's a settings for the `biomage stage` command. If you do `biomage stage ... --auto`, it'll automatically assign a random name for your deployment, and confirm "Yes" for the options.